appservice/src/intTest/groovy/nu/yona/server/EditGoalsTest.groovy

```diff
@@ -45,10 +45,6 @@ class EditGoalsTest extends AbstractAppServiceIntegrationTest
 		response.status == 200
 		response.responseData._links?.self.href == richard.url + appService.GOALS_PATH_FRAGMENT
 		response.responseData._embedded.budgetGoals.size() == 2
-		response.responseData._embedded.budgetGoals[1].activityCategoryName == 'gambling'
```

Comment:
Instead of removing it, it's better to still verify it:
```
response.responseData._embedded.budgetGoals.findAll{ it.activityCategoryName == 'gambling'}.size() == 1
```